### PR TITLE
Added simple documentation for bubble sort algorithm 

### LINKED ISSUE #444: Simple documentation for bubble sort algorithm
# [Title] 
[short description - you can make links to any proper, publicly available explanations of the algorithm.]

# Implementations
[add links to currently available implementation under the corresponding folder according to the languages]

### PR DESCRIPTION
**Pull request by zinthanmm**
**Hi!**  
Thanks for considering contributing to this ever-growing list of algorithm and data structure implementations.
Your contribution is valuable. Use the follow template for creating Pull Requests
---

**Fixes #444 

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

### CODE REVIEW

File: algorithms/ar-bsrt/README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+# Implementations
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
+- [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/java/BubbleSort.java)
+- [JavaScript](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/javascript/bubble_sort.js)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:18]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+[Learn more about Bubble Sort Algorithm](https://www.studytonight.com/data-structures/bubble-sort)
+
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+# Implementations
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:16]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+
+Bubble Sort is a simple sorting algorithm. This sorting algorithm is a comparison-based algorithm in which each pair of adjacent elements are compared and the elements are swapped if they are not in order.
+
+The below image illustrates a simple explanation of how the Bubble Sort Algorithm works. To learn more about Bubble Sort Algorithm refer to the below-mentioned link.
+
+[Learn more about Bubble Sort Algorithm](https://www.studytonight.com/data-structures/bubble-sort)
+
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+# Implementations
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:11]: Codacy found an issue: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
+- [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/java/BubbleSort.java)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:17]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 5]
```diff
@@ -1 +1,20 @@
...
+
+[Learn more about Bubble Sort Algorithm](https://www.studytonight.com/data-structures/bubble-sort)
+
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+# Implementations
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:15]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 6]
```diff
@@ -1 +1,20 @@
...
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
+- [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/java/BubbleSort.java)
+- [JavaScript](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/javascript/bubble_sort.js)
+- [Python 2](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/python2/bubble_sort.py)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:19]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 7]
```diff
@@ -1 +1,20 @@
...
+- [Python 3](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/python3/bubble_sort_exe.py)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:20]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

**Comment by zinthanmm** [comment]
Let me know if any changes need to be done.

File: algorithms/ar-bsrt/README.md [hunk 8]
```diff
@@ -1 +1,22 @@
...
+
+[Learn more about Bubble Sort Algorithm](https://www.studytonight.com/data-structures/bubble-sort)
+
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+## Implementations
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:17]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 9]
```diff
@@ -1 +1,22 @@
...
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
+- [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/java/BubbleSort.java)
+- [JavaScript](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/javascript/bubble_sort.js)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:20]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 10]
```diff
@@ -1 +1,22 @@
...
+- [Python 2](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/python2/bubble_sort.py)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:21]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 11]
```diff
@@ -1 +1,22 @@
...
+[Learn more about Bubble Sort Algorithm](https://www.studytonight.com/data-structures/bubble-sort)
+
+![alt text](https://i1.wp.com/gaebster.ch/wp-content/uploads/2018/10/bubblesort.png?ssl=1)
+
+## Implementations
+
+Below are some examples of how the Bubble Sort Algorithm is implemented using different programming languages.
+
+- [C](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/c/bubble_sort.c)
+- [C++](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/cpp/bubble_sort.cpp)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:18]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 12]
```diff
@@ -1 +1,22 @@
...
+- [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/java/BubbleSort.java)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:19]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

File: algorithms/ar-bsrt/README.md [hunk 13]
```diff
@@ -1 +1,22 @@
...
+- [JavaScript](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/javascript/bubble_sort.js)
+- [Python 2](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/python2/bubble_sort.py)
+- [Python 3](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-bsrt/python3/bubble_sort_exe.py)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/README.md:22]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318046)

**Comment by RavinduSachintha** [comment]
> Let me know if any changes need to be done.

There is a simple issue as you can see. For the validation system, we need to make lists in markdown files with **three spaces with the dash sign**.
eg:
- Test item (this is not accepted)
-   Test item (this is accepted)

**Comment by zinthanmm** [comment]
I have done the formatting changes. @RavinduSachintha

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
@all-contributors please add @zinthanmm for doc

**Comment by RavinduSachintha** [comment]
@zinthanmm great job. Thank you for your contribution. keep it up :+1: And make sure to star this repo :wink:

**Pull request merged**